### PR TITLE
Lesson 3

### PR DESCRIPTION
**Pull request by oksana-liseczkaya**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Об авторе</title>
 </head>
+
 <body>
     <h1>&#9996; Немного обо мне</h1>
-    <p>Меня зовут Оксана. Мне на данный момент 2<sup>1</sup> год. Родилась в небольшом городке и большую часть жизни прожила там. <br/>После школы я переехала в Минск и начала поиски себя. <strong>Мой девиз по жизни</strong>:</p>
+    <p>Меня зовут Оксана. Мне на данный момент 2<sup>1</sup> год. Родилась в небольшом городке и большую часть жизни
+        прожила там. <br />После школы я переехала в Минск и начала поиски себя. <strong>Мой девиз по жизни</strong>:
```
Comment (ghost) [inline about/index.html]: We have discussed `br` in another inline tag.

File: about/index.html [hunk 2]
```diff
@@ -1,28 +1,36 @@
...
+    </p>
     <blockquote>
-        <p><em>Если вы хотите сделать что-то великое в один прекрасный день,<br/> помните: один прекрасный день — это сегодня.</em></p>
+        <p><em>Если вы хотите сделать что-то великое в один прекрасный день,<br /> помните: один прекрасный день — это
```
Comment (ghost) [inline about/index.html]: See the previous comment.

File: about/index.html [hunk 3]
```diff
@@ -1,28 +1,36 @@
...
+                сегодня.</em></p>
     </blockquote>
-    <hr/>
+    <hr />
     <h2>&#128099; Почему я выбрала Frontend разработку?</h2>
-    <p>Так как я уже <mark>более двух лет</mark> работаю менеджером в продуктовой IT-компании, очень часто приходится принимать участие в<br/> разработке нового продукта. Хочу углубить свои знания в этом напралении и быть полезной в этой сфере. </p>
-     <hr/>
+    <p>Так как я уже <mark>более двух лет</mark> работаю менеджером в продуктовой IT-компании, очень часто приходится
+        принимать участие в<br /> разработке нового продукта. Хочу углубить свои знания в этом напралении и быть
```
Comment (ghost) [inline about/index.html]: See the previous comment.

File: index.html [hunk 4]
```diff
@@ -1,12 +1,148 @@
...
+        </tr>
+    </table>
+    <br />
+    <form action="/" method="get">
+        <fieldset>
+            <legend>My first form</legend>
+            <br />
+            <label for="name">Name</label>
+            <br />
+            <input type="text" id="name" name="Name" placeholder="Name" />
```
Comment (ghost) [inline index.html]: Should be required.

File: index.html [hunk 5]
```diff
@@ -1,12 +1,148 @@
...
+            <br />
+            <label for="email">Email</label>
+            <br />
+            <input type="email" id="email" name="Email" />
```
Comment (ghost) [inline index.html]: Should be required.

File: index.html [hunk 6]
```diff
@@ -1,12 +1,148 @@
...
+            <br />
+            <label for="password">Password</label>
+            <br />
+            <input type="password" id="password" name="Password" />
```
Comment (ghost) [inline index.html]: Should be required.

File: about/index.html (hunk 1 above)
Comment (oksana-liseczkaya) [reply about/index.html]: replace tag br in tag p

File: about/index.html (hunk 2 above)
Comment (oksana-liseczkaya) [reply about/index.html]: replace tag br in tag p

File: about/index.html (hunk 3 above)
Comment (oksana-liseczkaya) [reply about/index.html]: replace tag br in tag p

File: index.html (hunk 4 above)
Comment (oksana-liseczkaya) [reply index.html]: make name required

File: index.html (hunk 6 above)
Comment (oksana-liseczkaya) [reply index.html]: make password required

File: index.html (hunk 5 above)
Comment (oksana-liseczkaya) [reply index.html]: make email required

**Pull request merged**